### PR TITLE
Updated faq-structure

### PR DESCRIPTION
**Pull request by aliana17**

### CODE REVIEW

**Comment by arshadkazmi42** [comment]
@anchal7299 Looks like your fork is very old. Can you update your fork repo with latest commits of this repo.
You can check this on how to do that https://stackoverflow.com/a/7244456/3136282

**Comment by anchal7299** [comment]
ohk

File: content/docs/faq-structure.md [hunk 1]
```diff
@@ -59,16 +59,17 @@ components/
...
+यदि आप एक प्रोजेक्ट शुरू कर रहें हैं, तो फ़ाइल संरचना चुनने के लिए [पांच मिनट से अधिक समय न लगाएं।](https://en.wikipedia.org/wiki/Analysis_paralysis) 
+उपरोक्त किसी भी दृष्टिकोण को चुनें (या अपने स्वयं के साथ आएं) और कोड लिखना शुरू करें! आपके द्वारा कुछ वास्तविक कोड लिखे जाने के बाद, आप इसे वैसे भी पुनर्विचार करना चाहेंगे।
 
-If you feel completely stuck, start by keeping all files in a single folder. Eventually it will grow large enough that you will want to separate some files from the rest. By that time you'll have enough knowledge to tell which files you edit together most often. In general, it is a good idea to keep files that often change together close to each other. This principle is called "colocation".
+यदि आप पूरी तरह से फंसे हुए महसूस करते हैं, तो सभी फ़ाइलों को एक फ़ोल्डर में रखकर शुरू करें। आखिरकार यह काफी बड़ा हो जाएगा कि आप कुछ फ़ाइलों को बाकी हिस्सों से अलग करना चाहेंगे। उस समय तक आपको यह पर्याप्त ज्ञान होगा कि आप यह बता सकेंगे की आप किन फ़ाइलों को अक्सर एक साथ संपादित करते हैं। सामान्य तौर पर, फाइलों को रखना एक अच्छा विचार है जो अक्सर एक-दूसरे के करीब बदल जाते हैं। इस सिद्धांत को "कोलोकेशन" कहा जाता है।
```
Comment (arshadkazmi42) [inline content/docs/faq-structure.md]: `को अक्सर एक साथ संपादित करते हैं` => `पर अक्सर काम करते हैं`

how about this?

File: content/docs/faq-structure.md [hunk 2]
```diff
@@ -35,11 +35,11 @@ profile/
...
 ```
 
-The definition of a "feature" is not universal, and it is up to you to choose the granularity. If you can't come up with a list of top-level folders, you can ask the users of your product what major parts it consists of, and use their mental model as a blueprint.
+"फीचर" की परिभाषा सार्वभौमिक नहीं है, और यह आप पर निर्भर है कि आप किस विवरण का चयन करते हैं। यदि आप शीर्ष स्तर के फ़ोल्डरों की सूची के साथ नहीं आ सकते हैं, तो आप अपने उत्पाद के उपयोगकर्ताओं से पूछ सकते हैं कि इसमें कौन से प्रमुख भाग हैं, और उनके मानसिक मॉडल का ब्लूप्रिंट के रूप में उपयोग कर सकते हैं।
 
-#### Grouping by file type {#grouping-by-file-type}
+#### फ़ाइल प्रकार द्वारा समूहीकरण {#grouping-by-file-type}
 
-Another popular way to structure projects is to group similar files together, for example:
+परियोजनाओं के लिए एक और लोकप्रिय तरीका समान फ़ाइलों को एक साथ समूहित करना है, उदाहरण के लिए:
```
Comment (arshadkazmi42) [inline content/docs/faq-structure.md]: `परियोजनाओं` => `प्रोजेक्ट्स`

**Comment by anchal7299** [comment]
Hello Arshad
Is my review completed?

**Comment by arshadkazmi42** [comment]
@anchal7299 Looks like I missed this with my other mails. 
I just looked into it, looks like there is some issue with the PR, its showing alot of commits which should not be here.
Can you do the below steps
- `git remote add upstream https://github.com/reactjs/hi.reactjs.org.git`
- `git pull upstream master --rebase`
- `git push origin master --force`

It this does not work, we might have to open a new PR.

**Comment by arshadkazmi42** [comment]
Closing due to inactivity

**Pull request closed without merging**